src/charm.py

```diff
@@ -709,22 +729,26 @@ def _get_tracing_endpoint_info(self) -> str:
 
         return self.tracing.otlp_http_endpoint() or ""
 
-    def _update_kratos_endpoints_relation_data(self, event: RelationEvent) -> None:
-        logger.info("Sending endpoints info")
+    def _update_kratos_info_relation_data(self, event: RelationEvent) -> None:
+        logger.info("Sending kratos info")
 
-        admin_endpoint = (
-            self._admin_url
-            or f"http://{self.app.name}.{self.model.name}.svc.cluster.local:{KRATOS_ADMIN_PORT}"
-        )
-        public_endpoint = (
-            self._public_url
-            or f"http://{self.app.name}.{self.model.name}.svc.cluster.local:{KRATOS_PUBLIC_PORT}"
-        )
+        (admin_endpoint, public_endpoint) = self._kratos_endpoints
+        providers_cofigmap_name = "providers"
+        schemas_cofigmap_name = "identity-schemas"
+        configmaps_namespace = self.model.name
 
-        admin_endpoint, public_endpoint = (
-            admin_endpoint.replace("https", "http"),
-            public_endpoint.replace("https", "http"),
+        self.info_provider.send_info_relation_data(
+            admin_endpoint,
+            public_endpoint,
+            providers_cofigmap_name,
+            schemas_cofigmap_name,
+            configmaps_namespace,
         )
+
+    def _update_kratos_endpoints_relation_data(self, event: RelationEvent) -> None:
```

Comment:
Let's add a `TODO` to remove this (btw when are we going to remove this?)

Reply:
We should remove this once we switch to using `kratos-info` in login ui.
Comment added